src/Samples/dotnet6server/Properties/launchSettings.json

```diff
@@ -0,0 +1,28 @@
+﻿{
+  "iisSettings": {
+    "windowsAuthentication": false,
+    "anonymousAuthentication": true,
+    "iisExpress": {
+      "applicationUrl": "http://localhost:8113",
+      "sslPort": 44345
+    }
+  },
+  "profiles": {
+    "dotnet6Server": {
+      "commandName": "Project",
+      "dotnetRunMessages": true,
+      "launchBrowser": true,
```

Comment:
I don't think `"launchBrowser": true` is applicable to CoreWCF. I think it will just be a nuisance having a browser window open which can't show anything.